x/shield/client/cli/tx.go

```diff
@@ -512,3 +514,95 @@ $ %s tx shield withdraw-reimbursement <proposal id>
 	}
 	return cmd
 }
+
+// GetCmdStakingPurchase implements the command for purchasing Shield.
+func GetCmdStakingPurchase(cdc *codec.Codec) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "staking-purchase [pool id] [shield amount] [description]",
+		Args:  cobra.ExactArgs(3),
+		Short: "purchase Shield through staking",
```

Comment:
Obtain shield through staking CTK

Reply:
what purchase id do you mean

Reply:
I saw some previous comments mentioning purchase id. Nevermind